### PR TITLE
Direnc88 neenah

### LINKED ISSUE #228: Adding Neenah to the Map
Hi everyone! This issue is to track the addition of Neenah to the map, which can be found at:

- Longitude: -88.462609
- Latitude: 44.185819

### PR DESCRIPTION
**Pull request by direnc88**
added pin to github map. @githubteacher 
closes #228

### CODE REVIEW

**Comment by direnc88** [comment]
@githubteacher I cannot seem to figure out what I did wrong here or what I should be correcting.

File: direnc88Neenah.topojson [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "type": "Feature", "geometry": {"type": "Point", "coordinates": [-88.462609], [44.185819]}, "properties": {"username": "direnc88"} }
```
Comment (dvcv) [inline direnc88Neenah.topojson]: you need to take out the brackets "[", "]" from the longitude and latitude

**Comment by hollenberry** [comment]
@direnc88 , @dvcv's commits are accurate! You can look into his commit history to see the changes that he made, or even change your coordinates slightly for extra practice!

In either case, this is currently good to 🚢 if you'd like!

**Comment by hollenberry** [comment]
To be specific, the problem was just that you had added internal brackets. "[long],[lat]" as opposed to "[long, lat]". You were super close! 👍

**Pull request merged**